Docs: Add testing guide (#263)
aragon/aragon-court: readme.md

  <a href="https://travis-ci.com/aragon/aragon-court/"/>
</img>

## Project

#### 👩‍️ [Become an Aragon Court juror](https://anj.aragon.org)
Aragon Court is now live on Ethereum mainnet. You can become a juror by staking 10,000 ANJ.

#### ⚖ [Check out the Aragon Court Dashboard](https://court.aragon.org)

> The Aragon Court Dashboard is the central app where all dispute-related tools are available for jurors.

Reviewer: ```suggestion
The Aragon Court Dashboard is the central app where all dispute-related information is available for jurors.
```

#### 📚 [Read the User Guide](https://help.aragon.org/category/47-aragoncourt) 

> Read the user guide if you have any doubts about the protocol or the Aragon Court related tools

Reviewer: ```suggestion
Read the user guide if you have any doubts about the Aragon Court protocol, Court Dashboard, or related tools.
```

Even though Aragon Court could theoretically resolve any type of binary dispute, in its first deployments it will be used to arbitrate **Proposal Agreements.** These agreements require entities creating a proposal in an organization to agree to its specific rules around proposal creation, putting some collateral at stake that could be lost if the Court finds the proposal invalid.

## Deployed instances

#### Mainnet

The mainnet instance of Aragon Court is deployed at [`0xee4650cBe7a2B23701D416f58b41D8B76b617797`](https://etherscan.io/address/0xee4650cBe7a2B23701D416f58b41D8B76b617797#code)

#### Testing

> There are a few testing instances deployed of Aragon Court, please refer to the [testing guide](/docs/8-testing-guide) to have a better understanding about how to use these.

Reviewer: ```suggestion
There are a few testing instances deployed of Aragon Court, please refer to the [testing guide](/docs/8-testing-guide) to have a better understanding on using these.
```
